forc/src/cli/commands/test.rs

```diff
@@ -85,10 +85,11 @@ fn print_tested_pkg(pkg: &TestedPackage, test_print_opts: &TestPrintOpts) -> Res
             false => ("FAILED", Colour::Red),
         };
         info!(
-            "      test {} ... {} ({:?})",
+            "      test {} ... {} ({:?}) - (gas used: {})",
```

Comment:
```suggestion
            "      test {} ... {} ({:?}, {} gas)",
```
Maybe could be slightly more concise by putting time cost and gas cost together like this? This would look something like:
```
    test test_foo ... ok (4.3s, 1234 gas)
```